packages/database/src/migrations/20210318031303-UseTupaiaServiceInStrive-modifies-data.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const ENTITY_AGGREGATION_CONFIG = {
```

Comment:
I placed all "standard" tasks that need to be done when converting a project to `tupaia` service in this single migration file, to make future conversions easier